data/wanderer dire storm.txt

```diff
@@ -0,0 +1,136 @@
+
+ship "Dire Storm"
+	sprite "ship/dire storm"
+	attributes
+		category "Heavy Warship"
+		licenses
+			"Wanderer Military"
+		"cost" 40500000
+		"shields" 54128
+		"hull" 48962
+		"required crew" 72
+		"bunks" 237
+		"mass" 832
+		"drag" 13.9
+		"heat dissipation" 0.5
+		"fuel capacity" 816
+		"cargo space" 424
+		"outfit space" 864
+		"weapon capacity" 276
+		"engine capacity" 240
+		weapon
+			"blast radius" 340
+			"shield damage" 3400
+			"hull damage" 1700
+			"hit force" 5100
+	outfits
+		"Sunbeam" 2
+		"Thunderhead Launcher" 2
+		"Thunderhead Missile" 80
+		"Sunbeam Turret" 2
+		"Wanderer Anti-Missile" 2
+		
+		"Blue Sun Reactor"
+		"Large Biochemical Cell"
+		"Dark Storm Shielding" 2
+		"Bright Cloud Shielding"
+		"Wanderer Ramscoop" 2
+		"Fuel Pod" 2
+		
+		"Type 4 Radiant Thruster" 2
+		"Type 4 Radiant Steering" 2
+		"Hyperdrive"
+		
+	engine -98 160
+	engine 98 160
+	engine 0 289
+	engine -62 232
+	engine 62 232
+	gun -12 -156 "Sunbeam"
+	gun 12 -156 "Sunbeam"
+	gun -17 -134 "Thunderhead Launcher"
+	gun 17 -134 "Thunderhead Launcher"
+	turret -34 16 "Sunbeam Turret"
+	turret 34 16 "Sunbeam Turret"
+	turret -38 105 "Wanderer Anti-Missile"
+	turret 38 105 "Wanderer Anti-Missile"
+	drone 84 63
+	drone -84 63
+	drone -72 63
+	drone 72 63
+	explode "small explosion" 40
+	explode "medium explosion" 70
+	explode "large explosion" 50
+	explode "huge explosion" 20
+	"final explode" "final explosion large" 1
+	description "The Dire Storm is a deep space command, control and science vessel, designed to be one of the first ships through the Eye, to begin cataloging the dangers of the unknown space beyond, and to provide a secure, mobile headquarters for the warships that stay behind to cover the Wanderers' retreat. Endless banks of climate-controlled storage allow the dozen (or so) science labs to take hundreds of soil, water and even biological specimens from desolate worlds. Four drones allow remote, efficient harvesting and in an emergency can provide cover for a fleet of ships."
+	
+ship "Dire Storm" "Dire Storm (Passenger)"
```

Comment:
We already have a PR to add a much more appropriate Wanderer passenger ship: https://github.com/endless-sky/endless-sky/pull/4102

Reply:
The Centipede and the Riptide are not tier upgrades on the Bactrian.

Reply:
What does the Bactrian have to do with anything in this PR?

Reply:
We have long agreed that the Bactrain is too powerful even in T2, I don't see why adding a T2 version will help that.